h/static/styles/partials/_form-checkbox.scss

```diff
@@ -9,6 +9,12 @@
   margin-right: auto;
 }
 
+.form-checkbox--inline {
+  color: $grey-4;
+  border: none;
+  display: flex;
```

Comment:
Throwing `flex` into the mix makes things lay out nicely but it was a bit of a stab in the dark.

Reply:
I've seen flex used quite a lot in our css but I'm not an expert either! lol